kebechet/managers/version/version.py

```diff
@@ -387,9 +413,20 @@ def run(  # type: ignore
                     _LOGGER.error("Giving up with automated release")
                     return
 
-                changelog = self._compute_changelog(
-                    repo, old_version, version_identifier, version_file=changelog_file
-                )
+                try:
+                    changelog = self._compute_changelog(
+                        repo,
+                        old_version,
+                        version_identifier,
+                        changelog_smart,
+                        changelog_classifier,
+                        changelog_format,
+                        version_file=changelog_file,
+                    )
+                except ThothGlyphException as exc:
+                    _LOGGER.exception("Failed to generate smart release log")
+                    issue.comment(str(exc))
+                    issue.close()
 
                 # If an issue exists, we close it as there is no change to source code.
                 if not changelog:
```

Comment:
Will be the comment below put as well? I think there is a missing `return` statement on error.

Reply:
Fixed